.buildkite/pipelines/serverless_deployment/create_deployment_tag.yml

```diff
@@ -0,0 +1,33 @@
+## Creates deploy@<timestamp> tag on Kibana
+
+agents:
+ queue: kibana-default
+
+steps:
+  - label: "List potential commits"
+    commands:
+      - ts-node .buildkite/scripts/serverless/create_deploy_tag/release_events.ts --state initialize
```

Comment:
This `release_events.ts` is for managing the wizard state in a central place in the buildkite job.

Reply:
renamed to `release_wizard_messaging.ts`